source/screens/FormCaseScreen.js

```diff
@@ -18,12 +18,16 @@ const FormCaseScreen = ({ route, navigation, ...props }) => {
   const [form, setForm] = useState(undefined);
   const [formQuestions, setFormQuestions] = useState(undefined);
   const [initialCase, setInitialCase] = useState(undefined);
-  const { caseData, caseId } = route && route.params ? route.params : {};
+  const { caseData, caseId, signMode } = route && route.params ? route.params : {};
```

Comment:
`signMode` was confusing for me at first. Could we change it to `isSignMode` to keep a boolean naming convention?